styles/variables-silicon.less

```diff
@@ -176,39 +176,51 @@
 
 // Input
 // ---------------------------------------
-@agate-input-border-width: 3px;
+@agate-input-decorator-border-bottom-width: 3px;
 @agate-input-decorator-border-width: 0;
 @agate-input-decorator-focus-border-width: 0;
+@agate-input-decorator-has-value-opacity: 0.7;
```

Comment:
I don't think the name "decorator" is proper because it is not applied on decorator.